build_tools/packaging/how_to_do_release.md

```diff
@@ -0,0 +1,119 @@
+# How to make a release available
+
+The steps for the release are:
```

Comment:
We may want a warning here saying these instructions are only for release team members 🤔 
(permissions should be set up that way, but just to deter anyone casually browsing)